### DISCUSSION TITLE
apisix dashboard登录问题

### DISCUSSION THREAD

**Discussion by w568590301** [apache/apisix#5854] [General]
初次登录的时候会重新跳转到登录页面，需重新再点一次登录才能正常登录。

**Comment by tokers** [comment]
> 

@bzp2010 Could you check it out? Thanks!

**Reply by juzhiyuan** [reply]
Hi @w568590301, this issue is fixed in the latest Dashboard, please have a try :) 

https://apisix.apache.org/downloads

**Comment by w568590301** [comment]
Thanks so much for replying! Excuse me, how do I add our company name to the Powered By page?

**Reply by juzhiyuan** [reply]
Hi @w568590301, 

You could leave comments here https://github.com/apache/apisix/issues/487

If possible, you could add my WeChat **JuShaoyao** for further discussion around Dashboard :)  Today we have an online meeting https://github.com/apache/apisix/discussions/5873

**Comment by w568590301** [comment]
我部署了最新版apisix dashboard 2.9的版本，偶尔登录的时候还是会出现跳转的情况

**Reply by juzhiyuan** [reply]
see https://github.com/apache/apisix/discussions/5854#discussioncomment-1857588

Today will release 2.10.1.
